CHANGELOG/CHANGELOG-1.5.1.md

```diff
@@ -8,9 +8,7 @@
 
 - CSM 1.5.1 release specific changes. ([#582](https://github.com/dell/csm/issues/582))
 - Show volumes associated with the tenant from the k8s server. ([#408](https://github.com/dell/csm/issues/408))
-- CSI PowerScale documentation misses the privileges to user to use CSM replication. ([#516](https://github.com/dell/csm/issues/516))
 
```

Comment:
@Prabhu-Dell Is there a feature for the dellctl command created by Twee team? It should be included in the changelog.

Reply:
There was one, i recall seeing one earlier. However, I was looking for it and couldnt find